src/components/dashboard/button-container/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { usePagenation } from '@/src/hooks/usePagenation'
+
+import S from './buttonContainer.module.scss'
+import DashboardButton from '../dashboard-button'
+import PagenationButton from '../pagenation-button'
+
+const MOCK_DATA = [
+  { name: 'first', person: 'first' },
+  { name: '0', person: '손동희' },
+  { name: '1', person: '손동희' },
+  { name: '2', person: '안귀영' },
+  { name: '3', person: '장혁' },
+  { name: '4', person: '강나무' },
+  { name: '5', person: '안귀영' },
+  { name: '6', person: '장혁' },
+  { name: '7', person: '손동희' },
+  { name: '8', person: '안귀영' },
+  { name: '9', person: '장혁' },
+  { name: '10', person: '강나무' },
+  { name: '11', person: '안귀영' },
+  { name: '12', person: '장혁' },
+  { name: '13', person: '손동희' },
+  { name: '14', person: '안귀영' },
+  { name: '15', person: '장혁' },
+  { name: '16', person: '강나무' },
+  { name: '17', person: '안귀영' },
+]
+
+function DashboardButtonContainer() {
+  const { currPage, lastPage, currPageData, onClickPrevPage, onClickNextPage } =
+    usePagenation(MOCK_DATA, 6)
+
+  return (
+    <div className={S.container}>
+      {currPageData.map((data) =>
+        data.name === 'first' ? (
+          <DashboardButton key={data.name} type="addDashboard" />
+        ) : (
+          <DashboardButton
+            key={data.name}
+            type="moveDashboard"
+            dashboard={data}
+          />
+        ),
+      )}
```

Comment:
P3: 삼항 연산자 로직을 type props 안에서 해결하면 코드 수가 줄어들고 가독성이 좋을 것 같아요.

```suggestion
      {currPageData.map((data) =>
        data.name === 'first' ? (
          <DashboardButton key={data.name} type="addDashboard" />
        ) : (
          <DashboardButton
            key={data.name}
            type="moveDashboard"
            dashboard={data}
          />
        ),
      )}
```

Reply:
감사합니다 훨씬 낫네요 수정했습니다!